### ISSUE TITLE
Suggestion Box?

### ISSUE THREAD

**Issue by aubergine10** [DoESLiverpool/somebody-should#483]
Inspired by #482, I was wondering if there's a suggestion box for ad-hoc feedback from DoES patrons? Often people have ideas and if there's nowhere to suggest them they are lost opportunity to improve things. A suggestion box would facilitate an easy way to gather ad-hoc feedback, and could be checked once a week with `somebody should` tickets created for any suggestions.

Should be relatively easy to construct one with laser cutter, and could have email address mentioned on it as that's not very well publicised at present.

**Comment by aubergine10** [comment]
I'd love to make one but my brain firmware is incompatible with 3D modelling.

Assuming the enclosure is something that can easily be opened and tinkered with, I'd be interested in creating a beacon + sensor combo to a) make people aware of its existence, and b) alert DoES team when there's something lurking in the box... I have a puck.js on it's way to me via the internets so once that arrives (next week-ish) I should be able to make a start on that.

**Comment by ajlennon** [comment]
There are apparently various online tools which you give dimensions and they return laser-cut files for fit-together boxes (!!) 

e.g. http://makeabox.io

**Comment by amcewen** [comment]
Indeed.  And there's a list of some of them (hidden) on the [laser-cutter and Inkscape](https://github.com/DoESLiverpool/wiki/wiki/LaserCutterInkscape#handy-extensions) page of the wiki.

/me edits that to add @ajlennon's suggestion, as it isn't on there at present

**Comment by aubergine10** [comment]
I was introduced to some 3D software on Thursday which is much more in-line with how my brain works, so I'll be having a go at this in a few weeks.

**Comment by DoESsean** [comment]
FY all your I's, I still think this is an excellent idea, and one that we should implement.

I suppose I'm just leaving this message as a memory jogger.

**Comment by DoESsean** [comment]
Anyone up for making a box? 

I actually do think this is a good idea, could even just be a good old fashioned wood-and-nails box, that would work.

**Comment by DoESsean** [comment]
I used the tool @ajlennon suggested, and got a PDF. If anyone knows how to turn this into a box, please do.
[makeabox-190.00W-240.00H-40.00D-3.00T-20190822155913.pdf](https://github.com/DoESLiverpool/somebody-should/files/3530967/makeabox-190.00W-240.00H-40.00D-3.00T-20190822155913.pdf)
